draft-packages/loading-spinner/KaizenDraft/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -77,3 +78,11 @@ export const LoadingSpinner = ({
     )}
   </div>
 )
+
+/**
+ * @deprecated draft-packages LoadingSpinner is deprecated. Please use Button from "@kaizen/loading-spinner" instead.
```

Comment:
Hmm...shouldn't this still go above the component itself instead of this wrapper?

Reply:
vs
![Screen Shot 2022-02-17 at 1 30 51 pm](https://user-images.githubusercontent.com/36558508/154393244-12a5713b-de06-43d7-bbe3-44dd6bcd70ce.png)

Reply:
Seemed strange me but I opted to go with this method since it provided the most feedback to the consumer - happy for recommendations otherwise :) @HeartSquared

Reply:
Also, "Please use Button"—copy pasta? 😄

Reply:
hahahha - yep! well I needed another commit to fix my moai issue!

Reply:
All good :) Although it makes me wonder if we could have the docblock as part of the HOC 🤔

Reply:
A Docblock in a HOC - Sounds almost Doctor Seuss-esque 😆 We could though, however there are instances where we'll still have to add them in separately like in Button that doesn't used the withDeprecatedComponent

Reply:
Of course, but those should be more rare instances so it should be okay :)
Although now that I think of it, can we actually put a param from the HOC in the comment? 🤔 Maybe it's not actually possible to do

Reply:
I'd be keen to try just not within this PR, since we'd be changing the `withDeprecatedComponent` :) Might create a separate ticket to investigate we can go back in an make an amendment to the deprecated files